pkg/controllers/management/node/controller.go

```diff
@@ -67,6 +67,7 @@ var aliases = map[string]map[string]string{
 	"otc":           map[string]string{"privateKeyFile": "privateKeyFile"},
 	"packet":        map[string]string{"userdata": "userdata"},
 	"vmwarevsphere": map[string]string{"cloudConfig": "cloud-config"},
+	"google":        map[string]string{"authEncodedJson": "authEncodedJson"},
```

Comment:
note: this won't do anything at the moment. @dramich mentioned long term goal is to add a driver so can keep it as is